src/main/thrift/parquet.thrift

```diff
@@ -810,9 +803,13 @@ struct ColumnMetaData {
   /** optional statistics for this column chunk */
   12: optional Statistics statistics;
 
-  /** Set of all encodings used for pages in this column chunk.
+  /**
+   * DEPRECATED: use is_fully_dict_encoded instead
```

Comment:
I would suggest making this a separate PR, I think we'd prefer to keep the changes as small and focused as possible?

Reply:
Agreed. I will create another PR for the Statistics change alone if we are OK merging that now.